api/app/signals/apps/signals/models/area.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import F, Max, Min
+from django.contrib.gis.db import models
+
+# TODO: slugfields for the "code" fields below
+
+
+class AreaManager(models.Manager):
+    # Only show the most recent properties for an area
+    def get_queryset(self):
+        qs = super().get_queryset().annotate(
+            updated_at=Max('properties__created_at'),
```

Comment:
`updated_at` based on `created_at` is this correct?

Reply:
Idea was that the oldest instance of `AreaProperties` corresponds to creation, and the newest one to the last update.